pipeline/processor/postprocessor.py

```diff
@@ -576,11 +578,11 @@ def _group_list_elements(self, contents: dict) -> dict:
                                     text_block,
                                     "possibly_contiguous_with_previous_page_list",
                                 )
-                            else:
-                                text_block = self._update_contiguity_attributes(
-                                    text_block,
-                                    "possibly_contiguous_with_same_page_context",
-                                )
+                        else:
+                            text_block = self._update_contiguity_attributes(
+                                text_block,
+                                "possibly_contiguous_with_same_page_context",
+                            )
```

Comment:
Noticed a mistake where I wasn't appending this metadata at the right tabulation level, so it sometimes wasn't being included. It's not really being used in the code as is though, so not really anything to worry about.